### ISSUE TITLE
Default export in the generated packages.d.ts

### ISSUE THREAD

**Issue by piotrpospiech** [zodern/meteor-types#1]
I added `zodern:types` to the `meteor-universe-i18n@2.0.0-rc.1`, and I ran into one problem. Generated `packages.d.ts` doesn't support `default export`. 

It should be checked if file has default export in it, and if so, `packages.d.ts` could have something like this:
```ts
export { default }  from 'package-types/${name}/package/${finalTypesPath}'
```

**Comment by zodern** [comment]
Thanks for reporting this. I published zodern:types@1.0.8-beta.1.  today that should fix it.

I would like to avoid having to read the type declaration files as long as possible. It seems typescript's old syntax for imports/exports is able to re-export everything, so it now uses that: https://github.com/zodern/meteor-types/blob/554b754123e78acbf655144c4ed05f43e0debb12/writer.js#L152-L155.

**Comment by zodern** [comment]
I released version 1.0.8 today along with a number of other bug fixes.
